src/main/java/pt/lighthouselabs/obd/commands/ObdCommandResult.java

```diff
@@ -0,0 +1,137 @@
+package pt.lighthouselabs.obd.commands;
+
+import android.os.Parcel;
+import android.os.Parcelable;
```

Comment:
This is a pure Java library. If you want to customize it for Android, you should extend it in `android-obd-reader`.